.gitignore

```
@@ -2,7 +2,11 @@
 .env
 
 # Development files
-bin/
-group_vars/
+bin/*
+!bin/ci
```

The following suggestion makes more sense to me:

```suggestion
bin/arnold
bin/init-cluster
```

More importantly, I think the Makefile should be modified to not delete the `bin/` directory!

---

Both have been fixed sir.